src/pubsub/mod_pubsub_db_rdbms_sql.erl

```diff
@@ -381,13 +71,6 @@ insert_pubsub_node(Key, Name, Type, Owners, Options) ->
     EscOptions = esc_string(Options),
     sql_node_insert(EscKey, EscName, EscType, EscOwners, EscOptions, RDBMSType).
 
-update_pubsub_node(Nidx, Type, Owners, Options) ->
-    EscNidx = esc_int(Nidx),
-    EscType = esc_string(Type),
-    EscOwners = esc_string(Owners),
-    EscOptions = esc_string(Options),
-    sql_node_update(EscNidx, EscType, EscOwners, EscOptions).
-
 sql_node_insert(EscKey, EscName, EscType, EscOwners, EscOptions, {odbc, mssql}) ->
```

Comment:
My suggestion is to just execute an insert followed by a select. Other option might be to allow this optimized INSERT+SELECT as a simple query, but I think this should be done separately, not in this PR.